### DISCUSSION TITLE
Fixed number of rows/weeks

### DISCUSSION THREAD

**Discussion by dominicarrojado** [Hacker0x01/react-datepicker#3801] [Ideas]
May can keep the number of weeks or rows per month to be "6". This way the height of the datepicker will not change and will be fixed on all months.

 ![image](https://user-images.githubusercontent.com/6789191/197489673-bfcf0a2f-9587-42a1-b5e7-2295c42c3aba.png) ![image](https://user-images.githubusercontent.com/6789191/197489643-f00e32af-ba11-4f3f-ab82-0ef3119693eb.png)

**Comment by Jayeshagarwala** [comment]
use: 
`fixedHeight` in datepicker property
